src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -34,57 +34,34 @@ To create a service principal you need:
 
 #### 2. Role definition
 
-Download our [role definition](https://raw.githubusercontent.com/giantswarm/azure-operator/master/policies/guest.json).
+Download our [role definition template](https://raw.githubusercontent.com/giantswarm/azure-operator/master/policies/guest.json). Open it and replace `${SUBSCRIPTION_ID}` with your subscription ID.
 
-Open it and replace `${SUBSCRIPTION_ID}` with your subscription id.
+To find out your subscription ID you can use [the Azure portal](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade), as shown in the screenshot below:
 
-You can find it:
-
-- [here](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade) using the portal
 ![Azure Subscriptions list](/img/azure-subscriptions-list.png)
 
-- using Azure cli
-```
+Alternatively you can use the Azure CLI like so:
+
+```nohighlight
 $ az account list --output table
 Name                         CloudName    SubscriptionId                        State    IsDefault
 ---------------------------  -----------  ------------------------------------  -------  -----------
 Example                      AzureCloud   6ec148b8-8bea-4dd3-82bc-1787c8260e4a  Enabled  True
 ```
 
-Create the role definition
+With the edited role definition in the file `guest.json`, create the role definition using the Azure CLI:
 
-```
+```nohighlight
 $ az role definition create --role-definition @guest.json
-{
-  "assignableScopes": [
-    "/subscriptions/6ec148b8-8bea-4dd3-82bc-1787c8260e4a"
-  ],
-  "description": "Role for github.com/giantswarm/azure-operator",
-  "id": "/subscriptions/6ec148b8-8bea-4dd3-82bc-1787c8260e4a/providers/Microsoft.Authorization/roleDefinitions/70969f8f-eab5-4db7-87bf-5c8053431a85",
-  "name": "70969f8f-eab5-4db7-87bf-5c8053431a85",
-  "permissions": [
-    {
-      "actions": [
-        "*"
-      ],
-      "dataActions": [],
-      "notActions": [
-        "Microsoft.Authorization/elevateAccess/Action"
-      ],
-      "notDataActions": []
-    }
-  ],
-  "roleName": "azure-operator",
-  "roleType": "CustomRole",
-  "type": "Microsoft.Authorization/roleDefinitions"
-}
 ```
 
+TODO: explain what to do with the JSON output, or explain that the output can be discarded.
```

Comment:
We could mention here that `"roleName": "azure-operator"` is to be used in the next step for the role argument (`--role "azure-operator"`). So we are safe in the eventuality that we update the role name in our template file.

Reply:
Hmm. That name `azure-operator` is coming from our own template, isn't it? If yes, then I don't see a need to mention it here again.

Reply:
Ok